.travis.yml

```diff
@@ -1,9 +1,19 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-    - linux
-    - osx
+  - linux
+  - osx
 julia:
-    - 0.5
-    - nightly
+  - 0.7
+  - nightly
+# matrix:
+#   allow_failures:
+#     - julia: nightly
+
 notifications:
-    email: false
+  email: false
+after_success:
+  # push coverage results to Coveralls
+  - julia -e 'cd(Pkg.dir("FunctionalCollections")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```

Comment:
This doesn't actually work in 0.7 because of `Pkg.dir`. I'd suggest maybe
```yaml
- |
    julia -e "
        using Pkg
        Pkg.add("Coverage")
        using Coverage
        cd(\"$TRAVIS_BUILD_DIR\")
        Coveralls.submit(Coveralls.process_folder())
    "
```
but I'm not certain that will work if the tests are run somewhere other than the directory into which Travis cloned the package.

Reply:
It does actually work on Travis. I'm not sure the mechanism, but `Pkg.dir()` works and throws a warning on Travis in v0.7, rather than just erroring out. `PkgDev.generate()` also still generates calls like this, even on 0.7, so I think this is OK to keep for now.

Reply:
See: https://travis-ci.org/JuliaCollections/FunctionalCollections.jl/jobs/406699583#L613